server/routes/index.ts

```diff
@@ -216,6 +217,7 @@ export default function routes(services: Services): Router {
   router.use(csraRouter(services))
   router.use(appointmentRouter(services))
   router.use(professionalContactsRouter(services))
+  router.use(goalsRouter(services))
```

Comment:
I'm not sure whether this is the right pattern or not (to use a controller class like this) - all the other examples below (including the W&S page itself) use that inline syntax

Personally I prefer a controller class because you can unit test it - but I wonder why there are 2 patterns/approaches here, and which we should be following. Is it worth asking on the connect-dps-devs channel?

Reply:
The team have agreed with the approach taken in this PR.